pkg/descheduler/node/node.go

```diff
@@ -86,6 +86,7 @@ func GetNodeLister(client clientset.Interface, stopChannel <-chan struct{}) core
 	return nodeLister
 }
 
+// IsReady checks if the descheduler could run against give node.
```

Comment:
nit: just noticed so you might want to rephrase this "give node"?

Reply:
Done. Thanks for catching it.